wideq/core.py

```diff
@@ -188,49 +75,84 @@ def lgedm_post(url, data=None, access_token=None, session_id=None):
     authenticated requests. They are not required, for example, to load
     the gateway server data or to start a session.
     """
+
+    _LOGGER.debug("lgedm_post before: %s", url)
```

Comment:
Debugging stuff like this can probably be left out.

Reply:
I see that you implement a different type of logging, so maybe the log have to be reviewed?

Reply:
Sure, sounds good—let's leave that for a future pull request.